icu4c/source/i18n/measunit.cpp

```diff
@@ -547,9 +517,7 @@ static const char * const gSubTypes[] = {
     "solar-mass",
     "stone",
     "ton",
-    "", // TODO(ICU-21076): manual edit of what should have been generated by Java.
-    "percent", // TODO(ICU-21076): regenerate, deal with duplication.
-    "permille", // TODO(ICU-21076): regenerate, deal with duplication.
```

Comment:
These fall under "concentr". I expect they were added under "none" when we didn't have the "concentr" type yet.